Services/CustomDeviceModels.cs

```diff
@@ -42,16 +42,20 @@ public interface ICustomDeviceModels
     public class CustomDeviceModels : ICustomDeviceModels
     {
         private const string STORAGE_COLLECTION = "deviceModels";
+        private const string SERVICE_ERROR_EVENT = "ServiceError";
```

Comment:
I see the same constant is present in multiple files, it will make the code easier to maintain if we group all the diagnostics constants in one place. For instance I would create a `DiagnosticsConstants.cs` file in the `Diagnostics` folder with this any other value. e.g.

```
public class DiagnosticsConstants
{
    public const string SERVICE_ERROR_EVENT = "ServiceError";
}

Reply:
Anyhow, as per other comment about using a semantic log approach, I would actually move this constant (and keep it private) inside the logger class.